api-reference/testrpc/README.md

```diff
@@ -0,0 +1,23 @@
+# Echo TestRPC protocol
+
+Echo TestRPC provides separate executable designed to make testing faster and easier.
+
+* Transactions are applied instantly and have no cost.
+* Simplified block generation mechanism
+* Accounts can be re-cycled, reset and instantiated with a fixed amount of any asset.
+* Chain parameters and gas price can be modified.
+
+Echo TestRPC API is based on Ethereum TestRPC protocol and extends JSON API.  
+More details on the API can be found on [TestRPC API page](methods.md).
+
+The new block generation mechanism generates a new block every 5 seconds if any new transaction presents. This period can be changed in config. Echorand consensus and block generator are disabled in TestRPC.
```

Comment:
Can we generate a block immediately upon the arrival of a new transaction? Waiting for even a few seconds can increase test execution time.

Reply:
Yes. Block generation period can be set to zero by default.